meta-mbl-distro/recipes-core/packagegroups/packagegroup-mbl-development.bb

```diff
@@ -24,6 +24,8 @@ PACKAGEGROUP_MBL_DEVELOPMENT_PKGS_append = " glib-2.0"
 PACKAGEGROUP_MBL_DEVELOPMENT_PKGS_append = " python3-pygobject"
 PACKAGEGROUP_MBL_DEVELOPMENT_PKGS_append = " gobject-introspection"
 PACKAGEGROUP_MBL_DEVELOPMENT_PKGS_append = " e2fsprogs"
+PACKAGEGROUP_MBL_DEVELOPMENT_PKGS_append = " mbed-crypto"
```

Comment:
Shouldn't `mbed-crypto` be included in the production packagegroup?

Reply:
I don't think so - I don't think we should be including a library in the production packagegroup when nothing actually uses it. If somebody needs mbed-crypto then it should probably be added to their image via a dependency of a recipe that uses it.

When mbed-crypto becomes a dependency of the cloud client then it will be pulled into the production image via that dependency.

Reply:
...perhaps by the same logic I shouldn't have included "mbed-crypto" in the dev package group - I should just have included "mbed-crypto-test".

Reply:
https://github.com/ARMmbed/meta-mbl/pull/576/commits/b4e6d736b258d6d2185173d4d31401d3ad28a224